concourse/scripts/test_and_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -l
+
+set -exo pipefail
+
+function _main() {
+    # FIXME: The test db and extension creation should be handled by python code.
+    createdb gpadmin
+    psql gpadmin -c "CREATE LANGUAGE plpython3u;"
+
+    # Run testing
+    pushd /home/gpadmin/greenplumpython_src
+    unset PYTHONPATH
+    unset PYTHONHOME
+    tox -e test_py39
+    # build wheel
+    pip3 wheel . -w ../greenplumpython_artifacts
+    popd
+    tar -czf greenplumpython_artifacts/greenplumpython.tar.gz greenplumpython_artifacts/*.whl
```

Comment:
So user needs to install `psycopg2` manually?

Reply:
what will be the full name of the wheel file on different platforms? On my arch, it is:
```
greenplum_python-1.0.0b2-py3-none-any.whl
```

Reply:
can we do a simple test here? Just install the wheel and do a `import` test.